attic/rfc_metrics.md

```diff
@@ -36,14 +51,55 @@ Internally, the function would look the following:
 
 Example:
 ```r
-fn = function(truth, response, positive, ...) {
-  assert_binary(truth, response = response, positive = positive)
-  fn_cm(cm(truth, response, positive))
-}
+MeasureFairness = R6Class("MeasureFairness", inherit = Measure, cloneable = FALSE,
+  public = list(
+    fun = NULL,
+    na_value = NaN,
+    base_measure = NULL,
+    #' @description
+    #' Creates a new instance of this [R6][R6::R6Class] class.
+    initialize = function(name, base_measure) {
+      info = mlr3fairness::measures[["groupwise_abs_diff"]]
```

Comment:
This implies that there is a 'list` or environment `measures` that lives in mlr3fairness and contains this measure.
This is not stated anywhere in the RFC. 
So we should either also define this measure or (which is likely) not use it at all and instead just define the relevant info in this class.

Also: measures in general are stored in `mlr_measures` and can be accessed via `msr(...)`. So why not make use of this here if required?